code/game/gamemodes/events/powercreeper.dm

```diff
@@ -138,6 +141,8 @@
 /obj/structure/cable/powercreeper/proc/try_electrocution(var/mob/living/M)
 	if(!istype(M) || M.isDead())
 		return 0
+	if(M_NO_SHOCK in M.mutations)
```

Comment:
this try_electrocution proc should probably be
```dm
if(electrocute_mob(M, powernet, src)
   blablabla
```

this mutation check should be inside electrocute_mob. If I'm not mistaken there are two electrocution procs, we should merge them

Reply:
This is the mob oriented one. The other proc is for a turf, and it shocks everything on that turf by calling this proc. In other words, they are as merged as they can be for their different purposes.

Reply:
electrocute_mob is the general proc not tied to any atom (you can find it in powernet.dm)

so this check could be moved there, but that's the extent of it, we don't want to move anything else to that proc because electrocute_mob doesn't do a beam effect and can effect dead mobs, etc. (it's used for all electrocutions)